check_rooted.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+def check_rooted
+  # use capture3 to get error printed also
+  _, stderr_str, = Open3.capture3('adb shell sudo')
```

Comment:
THis won't work for all cases:

1. sometimes OS exposes `su`, but not `sudo`
2. some ROMS dump user right into root, so `su` and `sudo` won't do anything

Solution is `adb shell whoami == root`